packages/core/next.config.js

```diff
@@ -6,6 +6,7 @@ const storeConfig = require('./faststore.config')
  * */
 const nextConfig = {
   /* config options here */
+  output: 'standalone',
```

Comment:
nice!  This makes us compliant with the cicd platform config (that ensures standalone mode), and improves our local tests.

curiosity: are there other advantages?

Reply:
The big advantage of the standalone build is the small bundle size sent to pipeline, Next.js only considers the required files (including `node_modules`) for building the store and does the magic with the small sizing of standalone build directory  👌🏼

Reply:
I mean, we already set standalone in the platform. But setting here keeps the local build "the same" (standalone) as the platform.

Reply:
I think that we shouldn't have this config set here. If someone wants to host their store on Vercel they should be able to use its full features. Forcing standalone is a restriction from our hosting infra